heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -163,17 +165,22 @@ public boolean isEmpty() {
             final AggregationInstance aggregation = request.getAggregation();
             final DateRange range = request.getRange();
             final QueryContext queryContext = request.getContext();
+            final Features features = request.getFeatures();
 
             queryLogger.logIncomingRequestAtNode(queryContext, request);
 
-            final boolean slicedFetch = request.getFeatures().hasFeature(Feature.SLICED_DATA_FETCH);
+            final boolean slicedFetch = features.hasFeature(Feature.SLICED_DATA_FETCH);
+
+            final BucketStrategy bucketStrategy =
+                options.getBucketStrategy().orElse(BucketStrategy.END);
```

Comment:
There's no other bucket strategy right now. The new strategy and that check will be introduced in a separate PR.